src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -1189,8 +1202,7 @@
 examples:
   - name: Validate whether a template is syntactically correct. (autogenerated)
     text: |
-        az group deployment validate --parameters "{ \\"location\\": { \\"value\\": \\"westus\\" } }" \\
-            --resource-group MyResourceGroup --template-file storage.json
+        az group deployment validate --parameters "{ "location": { "value": "westus" } }"      --resource-group MyResourceGroup --template-file storage.json
```

Comment:
> "{ \\"location\\": { \\"value\\": \\"westus\\" } }" 

In fact, the `\\` in JSON is used to escape `"`. I think it should be retained, otherwise users will get the JSON format errors when using this example.

Reply:
agree with xing

Reply:
reverted